test/test_evaluation/test_train_evaluator.py

```diff
@@ -3166,6 +3294,38 @@ def test_eval_holdout_budget_subsample(self):
         self.assertEqual(info[0]["status"], StatusType.SUCCESS)
         self.assertNotIn("bac_metric", info[0]["additional_run_info"])
 
+    def test_eval_holdout_budget_subsample_multi_objective(self):
+        metrics = {
+            accuracy: 0.0,
+            balanced_accuracy: 0.0,
```

Comment:
[Q]
We get successful configurations but `0.0` on both metrics?

Reply:
Apparently yes. The predictions appears to be perfect in this case.

Reply:
Ahh these are losses, that makes sense